common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -809,3 +809,23 @@ def test_perform_problem_reset_and_navigate(self):
         self.go_to_tab_and_assert_problem(1, self.problem1_name)
         problem1_content_after_coming_back = self.problem_page.problem_content
         self.assertEqual(problem1_content_before_switch, problem1_content_after_coming_back)
+
+    def test_perform_problem_reset_and_focus(self):
+        """
+        Scenario:
+        I go to sequential position 1
+        Facing problem1, I select 'choice_1'
+        Then perform the action reset
+        Observe that the focus has shifted to the expected element
+        """
+        # Go to sequential position 1 and assert that we are on problem 1.
+        self.go_to_tab_and_assert_problem(1, self.problem1_name)
+
+        # Update problem 1's content state – by performing reset operation.
```

Comment:
I am not sure I am following you?  I am still working on the content in here so it may be worth waiting to review this PR.